frontend/packages/console-shared/src/sorts/nodes.ts

```diff
@@ -0,0 +1,14 @@
+import * as UIActions from '@console/internal/actions/ui';
+import { NodeKind } from '@console/internal/module/k8s';
+
+export const nodeMemory = (node: NodeKind): number => {
+  const free = Number(UIActions.getNodeMetric(node, 'freeMemory'));
+  const total = Number(UIActions.getNodeMetric(node, 'totalMemory'));
+  return (total - free) / total;
```

Comment:
We need to handle missing metrics.

Reply:
ack

Reply:
getNodeMetrics handles that. We return 0 and use that for sorting purposes.

Reply:
> getNodeMetrics handles that. We return 0 and use that for sorting purposes.

If we return 0, we can have a divide by 0 error here.

Reply:
@spadgett don't we get an `Infinity` if we divide by zero?